study.md

```diff
@@ -34,13 +34,13 @@ In ActiveRecord Migrations, what is the name of the method the creates a new
 table?
 
 ```md
-<!-- your response here -->
+bin/rails generate migration createtable
 ```
 
 What is the name of the method that creates a new column?
 
 ```md
-<!-- your response here -->
+bin/rails generate migration AddColumnToTable
```

Comment:
We want the Rails method that's used to create a new table column and it's `add_column`.